### DISCUSSION TITLE
2 brief CheckMATE2 analysis question regarding 13TeV analysis & the Total p+ p+ cross section

### DISCUSSION THREAD

**Discussion by AdamKimForster** [CheckMATE2/checkmate2#10] [Q&A]
Deear CheckMATE team.

Firstly, thanks for the super useful software :)

I have 2 brief question:

1. I have been running a 13TeV analysis (see attached .txt file). However, when I look at the Pythia log files they say "We collide p+ with p+ at a CM energy of 8.000e+03 GeV” (see line 154 of attached .log file). Am I doing something wrong?

2. I am looking for an output of the total p+ p+ cross section. I have found some references to cross sections but it is not clear to me that they refer to p+ p+ and not to specific sub processes. Do you know how I can find the total p+ p+ cross section?

Any help would be greatly appreciated.

[cMSSMparam.txt](https://github.com/CheckMATE2/checkmate2/files/6231582/cMSSMparam.txt)

[pythia_allsusy.log](https://github.com/CheckMATE2/checkmate2/files/6231573/pythia_allsusy.log)
Best

Adam

**Comment by agagsgroove** [comment]
Hi Adam,

sorry for our late reply. It seems to me that there might be a bug in CM. Could you try atlas13TeV and/or cms13TeV, please?

Regarding the total cross section. It's given in the pythia log. You can "grep" for "| sum" and then should find it. It's given in units of millibarn. I have just copied the relevant part below.

Cheers,

Jong

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

*-------  PYTHIA Event and Cross Section Statistics  -------------------------------------------------------------*
 |                                                                                                                 |
 | Subprocess                                    Code |            Number of events       |      sigma +- delta    |
 |                                                    |       Tried   Selected   Accepted |     (estimated) (mb)   |
 |                                                    |                                   |                        |
 |-----------------------------------------------------------------------------------------------------------------|
 |                                                    |                                   |                        |
 | q qbar' -> ~chi_10 ~chi_10                    1551 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_10 ~chi_20                    1552 |        8180        822        822 |   5.021e-15  1.002e-16 |
 | q qbar' -> ~chi_20 ~chi_20                    1553 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_10 ~chi_30                    1554 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_20 ~chi_30                    1555 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_1+ ~chi_10                    1571 |       12238       1320       1320 |   8.236e-15  1.304e-16 |
 | q qbar' -> ~chi_1- ~chi_10                    1572 |        3744        286        286 |   1.828e-15  6.022e-17 |
 | q qbar' -> ~chi_2+ ~chi_10                    1573 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_2- ~chi_10                    1574 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_1+ ~chi_20                    1575 |       12176       1320       1320 |   8.236e-15  1.312e-16 |
 | q qbar' -> ~chi_1- ~chi_20                    1576 |        3688        276        276 |   1.895e-15  6.233e-17 |
 | q qbar' -> ~chi_2+ ~chi_20                    1577 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_2- ~chi_20                    1578 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_1+ ~chi_30                    1579 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_1- ~chi_30                    1580 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_1+ ~chi_1-                    1591 |        9470        976        976 |   5.994e-15  1.103e-16 |
 | q qbar' -> ~chi_1+ ~chi_2-                    1592 |           0          0          0 |   0.000e+00  0.000e+00 |
 | q qbar' -> ~chi_2+ ~chi_1-                    1593 |           0          0          0 |   0.000e+00  0.000e+00 |
 |                                                    |                                   |                        |
 | sum                                                |       49496       5000       5000 |   3.121e-14  2.529e-16 |
 |                                                                                                                 |
 *-------  End PYTHIA Event and Cross Section Statistics ----------------------------------------------------------*

**Reply by AdamKimForster** [reply]
Hi

Thanks very much for your help!

I have just tried cms and atlas runs and it seems the same problem is occurring (see attached files).

Best

Adam

[pythia_allsusy-cms.log](https://github.com/CheckMATE2/checkmate2/files/6242585/pythia_allsusy-cms.log)
[pythia_allsusy-atlas.log](https://github.com/CheckMATE2/checkmate2/files/6242587/pythia_allsusy-atlas.log)

**Reply by AdamKimForster** [reply]
I installed this version of checkmate a couple of years ago. I will try installing a more up to date version
